mem/mem_bsd.go

```diff
@@ -0,0 +1,87 @@
+// +build freebsd openbsd
+
+package mem
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"strconv"
+	"strings"
+)
+
+const swapCommand = "/sbin/swapctl"
+
+// swapctl column indexes
+const (
+	nameCol     = 0
+	totalKiBCol = 1
+	usedKiBCol  = 2
+)
+
+func SwapDevices() ([]*SwapDevice, error) {
+	return SwapDevicesWithContext(context.Background())
+}
+
+func SwapDevicesWithContext(ctx context.Context) ([]*SwapDevice, error) {
+	output, err := exec.Command(swapCommand, "-lk").Output()
```

Comment:
Could you use `exec.LookPath` like [here](https://github.com/shirou/gopsutil/blob/master/internal/common/common_freebsd.go#L31)?